helmfile/values/dex.yaml.gotmpl

```
@@ -14,26 +9,35 @@ ingress:
     nginx.ingress.kubernetes.io/whitelist-source-range: {{ .Values.externalTrafficPolicy.whitelistRange.dex }}
     {{ end }}
   tls:
-  - secretName: dex-tls
-    hosts:
-    - dex.{{ .Values.global.baseDomain }}
+    - secretName: dex-tls
+      hosts:
+        - dex.{{ .Values.global.baseDomain }}
   hosts:
-  - dex.{{ .Values.global.baseDomain }}
+    - host: dex.{{ .Values.global.baseDomain }}
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
 
 # https termination by ingress instead of through dex
-https: false
+https:
+  # -- Enable the HTTPS endpoint.
+  enabled: false
 
 {{- if .Values.dex.google.groupSupport }}
-extraVolumeMounts:
+volumeMounts:
 - mountPath: /etc/dex/google
   name: google-sa
-extraVolumes:
+volumes:
 - name: google-sa
   secret:
       secretName: {{ .Values.dex.google.SASecretName }}
 {{- end }}
 
 config:
+  storage:
+    type: memory
+    config:
+      inCluster: true
```

I think that those two lines are just for `type:kubernetes`
```suggestion
  storage:
    type: memory
```